applications/asset_tracker_v2/src/modules/modem_module.c

```diff
@@ -505,72 +487,15 @@ static void send_edrx_update(float edrx, float ptw)
 	APP_EVENT_SUBMIT(evt);
 }
 
-static inline int adjust_rsrp(int input, enum sample_type type)
+static inline int adjust_rsrp(int input)
```

Comment:
Yep, but they perform the conversion depending on a different Kconfig. When these were needed only in modem module, this Kconfig difference was handled by `enum sample_type`. Now that one version is needed in modem module, and the other one in location module. I'm not sure there would be a convenient place for such a utility function in current ATv2 architecture. So I thought just copying the function and having the different Kconfig check in different versions of the function is the cleanest solution. Kconfig check is anyway the only real functionality in the function because otherwise the caller's could just use `RSRP_IDX_TO_DBM` directly.